sequencing/ATACSeq/preprocessing/progressReport.sh

```diff
@@ -134,10 +129,17 @@ do
     fi
     
     if [ ! -s ${PEAKDIR}/MACS/BAMPE/${sampleName}.broadPeak.filt ]
+    then
+        echo -n "N" >> ${METADIR}/summariseSampleProcessingProgress.csv
```

Comment:
For the MACS3PeaksPE column, shouldn't this have a comma after the N? Otherwise it won't be comma separated from the final column of the csv?